common/app/conf/AudioFlagship.scala

```diff
@@ -0,0 +1,9 @@
+package conf
+
+object AudioFlagship {
+  val tagName: String = "news/series/todayinfocus"
+  val title: String = "Today in Focus"
+  val seriesId: String = "today-in-focus"
+  val description: String = "Listen to the story behind the headlines <br />\n for a deeper understanding of the news. <br />\n <strong>Every weekday with Anushka Asthana</strong>."
```

Comment:
fyi, Scala supports multiline strings with whitespace handling, like this:
```scala
val description: String = """Listen to the story behind the headlines <br />
                            |for a deeper understanding of the news. <br />
                            |<strong>Every weekday with Anushka Asthana</strong>.
                            |""".stripMargin
```

Reply:
useful to know